### ISSUE TITLE
swap <connection name> and <option> in rvc

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#205]
Hi @jjr840430,

Can you swap the <connection name> and <option> in `rvc` parameters please?

So now we have this:

```
$ rvc connect vpnname
$ rvc status vpnname
```

This means that when you want to see if the VPN is connecting you need to use the keyboard and go left left left left etc until you reach the `<option>` and backspace it and type in the new option. If the `<option>` is the last parameter then we can just quickly jump to the end of the line and replace it:

```
$ rvc vpnname connect
$ rvc vpnname status
```

Thanks

**Comment by jjr840430** [comment]
@erikbor It's very convenient way, Thanks.
I implemented it by #206

**Comment by erikbor** [comment]
Nice one thanks!!!
